src/ontology/cl-edit.owl

```diff
@@ -22687,6 +22689,16 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22527717"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_0019032 "intestinal tuft cell"@en)
 EquivalentClasses(obo:CL_0019032 ObjectIntersectionOf(obo:CL_0002204 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001277)))
 
+# Class: obo:CL_0300023 (respiratory hillock cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0001-7772-9578>) Annotation(oboInOwl:hasDbXref "PMID:30069044"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:32726565"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:33442032"^^xsd:string) obo:IAO_0000115 obo:CL_0300023 "An epithelial, transitional cell type between basal and secretory; located in stratified, non-ciliated structures with high cell turnover in respiratory epithelium. In some mammalian species, this cell type has been noted to express KRT13 and KRT4 and is postulated to play a role in squamous barrier function and immunomodulation."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_0300023 <http://orcid.org/0000-0002-1773-2692>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_0300023 "2022-06-06T08:31:09Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasNarrowSynonym obo:CL_0300023 "hillock cell of respiratory tract"^^xsd:string)
+AnnotationAssertion(rdfs:label obo:CL_0300023 "respiratory hillock cell"^^xsd:string)
+EquivalentClasses(obo:CL_0300023 ObjectIntersectionOf(obo:CL_4030024 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000065)))
+SubClassOf(obo:CL_0300023 obo:CL_0002368)
```

Comment:
I'm guessing this subclass axiom isn't needed - check if reasoner reasons this, if not check why?

Reply:
@shawntanzk, if I remove the subclass, the inferred axiom appears:
<img width="485" alt="Screenshot 2022-06-27 at 11 36 11" src="https://user-images.githubusercontent.com/95643797/175923113-8d0da62e-56c0-4b1c-b026-92a29a99e3f6.png">
But if I rerun the reasoner, it disappears:
<img width="485" alt="Screenshot 2022-06-27 at 11 36 26" src="https://user-images.githubusercontent.com/95643797/175923179-5d91e5d0-df0c-4f66-9225-f2ab57d42974.png">

The EA for 'respirator epithelial cell' is:
```
'endo-epithelial cell'
 and ('part of' some 'respiratory tract') 
```
Is it not inferred because 'endo-epithelial cell' is used instead of 'epithelial cell'? 
With that said, perhaps leave it as is?

Reply:
yes you are right, I just looked through it in protege and its gonna take some thought to get the inference to work. One way to deal with this would be to add ('develops from' some 'endodermal cell') to respiratory hillock cell, and convert respiratory epithelial cell to have equiv 
```
'endo-epithelial cell'
 and ('part of' some 'respiratory tract')
```
Both axioms are correct to add (I think) but I think this might cause even more difficulty in the future. 
I think you can leave it as is for now, sorry about that, I just always get a bit worried with equivalent classes with subclasses

Reply:
Will leave as is.